cypress/integration/apps/query_enhancements/constants.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const BASE_PATH = Cypress.config().baseUrl;
+export const DATASOURCE_NAME = 'test_cluster';
+export const DATASOURCE_URL = 'http://localhost:9200';
```

Comment:
We can update the DataSource_URL to use the     `openSearchUrl: 'http://localhost:9200'` defined within [this file ](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/cypress.config.ts#L24C1-L25C1) .

Reply:
will clean out all the base path and url in another follow-up